R/readStationary.R

```diff
@@ -1,37 +1,166 @@
 #' Load Stationary File as magclass object
 #'
-#' @return magclass object
+#' Reads in EDGE_TradMod.cs4r, which hast data for the SSP scenarios. In the convert function, the subset argument
+#' is used to specify the desired scenarios. Unspecified scenarios are copies of the SSP2 scenario.
 #'
-#' @author Antoine Levesque, Robin Hasse
+#' @return magclass object
+#' @order 1
 readStationary <- function() {
+  read.magpie("EDGE_TradMod.cs4r")
+}
 
-  scenarios <- list(SSPs  = paste0("SSP", 1:5),
```

Comment:
Did you talk about read- and convertStationaryy with @fbenke-pik ?

Reply:
No, ist there a something I should know @fbenke-pik ?

Reply:
it will be removed at some point, but for now we need to keep it.